tests/unit/amo/pages/TestAddon.js

```diff
@@ -883,6 +883,15 @@ describe(__filename, () => {
     ).toContain('<a>placeholder</a>');
   });
 
+  it('hides developers comments if null', () => {
```

Comment:
This is a good test case. Then we'll need at least one test case for when the add-on has a developer comment.